app/proguard-rules.pro

```diff
@@ -18,4 +18,9 @@
 
 # If you keep the line number information, uncomment this to
 # hide the original source file name.
-#-renamesourcefileattribute SourceFile
\ No newline at end of file
+#-renamesourcefileattribute SourceFile
+
+-dontwarn org.bouncycastle.jsse.**
```

Comment:
Is this a good idea? 🤔

Reply:
it just hides warnings from proguard, it shouldn't break anything